### ISSUE TITLE
The most efficient way to extend the list

### ISSUE THREAD

**Issue by mato75** [kamilkp/angular-vs-repeat#65]
When you have an expandable array, what would be the most sufficient way to extend array, without digesting the whole array over. 

     [].push.apply(scope.table, extending);

**Comment by FlavorScape** [comment]
Yeah I'm wondering if this is usable/efficient with real client-server paging in the background.
